Gordon360/Models/ViewModels/MembershipUploadViewModel.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+
+namespace Gordon360.Models.CCT
+{
+    public partial class MembershipUploadViewModel
+    {
+        public string Activity { get; set; }
+        public string Session { get; set; }
+        public string Username { get; set; }
+        public string Participation { get; set; }
+        public string CommentText { get; set; }
+        public bool GroupAdmin { get; set; }
+        public bool Privacy { get; set; }
+        public MEMBERSHIP ToMembership(int gordonId, DateTime beginDate)
+        {
+
+            return new MEMBERSHIP()
+            {
+                ACT_CDE = this.Activity,
+                SESS_CDE = this.Session,
+                ID_NUM = gordonId,
+                BEGIN_DTE = beginDate,
+                PART_CDE = this.Participation,
+                COMMENT_TXT = this.CommentText,
+                GRP_ADMIN = this.GroupAdmin,
+                PRIVACY = this.Privacy,
+                USER_NAME = Environment.UserName
+            };
+        }
```

Comment:
What is `Environment` here? I've never seen that before.

Reply:
See [this link](https://learn.microsoft.com/en-us/dotnet/api/system.environment.username?view=net-6.0)

Reply:
It is really System.Environment.UserName and it gets the username of the user who is currently executing the process/thread. Since the API is running as CCT.Service, I figured that we want to use that dynamically as the Jenzabar USER_NAME for changes.